internal/database/apiconfig.go

```diff
@@ -21,40 +21,47 @@ import (
 	"time"
 
 	"github.com/google/exposure-notifications-server/internal/ios"
-	"github.com/google/exposure-notifications-server/internal/model/apiconfig"
+	"github.com/google/exposure-notifications-server/internal/model"
 	"github.com/google/exposure-notifications-server/internal/secrets"
 )
 
+const (
```

Comment:
I actually prefer having the queries in the function for readability

the compiler output from this vs the previous way should be identical (string appears once in the output)

Reply:
Done